4788/src/main/include/Intake2.h

```diff
@@ -0,0 +1,79 @@
+#pragma once 
+
+#include "strategy/StrategySystem.h"
+#include "RobotMap.h"
+
+using actState = wml::actuators::BinaryActuatorState;
+
+enum class IntakeState {
+  IDLE,
+  MANUAL,
+  AUTO
+};
+
+enum class IntakeActuatorState {
+  DEPLOYED,
+  STOWED
+};
```

Comment:
You can combine these states:
```suggestion
enum class IntakeState {
  STOWED,      // Up, motor off
  DEPLOYED,   // Down, motor off
  INTAKING,    // Down, motor intake
  EJECTING     // Down, motor eject
};
```

This way you don't need a setpoint, since INTAKING and EJECTING both run at a fixed speed (you can define this in ControlMap). In this case, the idle/default state is STOWED.